codespeed/mercurial.py

```diff
@@ -45,8 +45,7 @@ def getlogs(endrev, startrev):
     updaterepo(endrev.branch.project, update=False)
 
     cmd = ["hg", "log",
-            "-r", "%s:%s" % (endrev.commitid, startrev.commitid),
-            "-b", "default",
+            "-r", "%s::%s" % (startrev.commitid, endrev.commitid),
```

Comment:
I would keep `endrev` first so that the last commit is shown first, and the rest in inverse order.